### ISSUE TITLE
Increase the font-size of the website to improve the user experience

### ISSUE THREAD

**Issue by SAY-droid427** [ocaml/v2.ocaml.org#1388]
Increase the font -size in the website. The current font-size makes it tough to read.
BEFORE:
![OcamlBefore1](https://user-images.githubusercontent.com/59281636/113478291-a8f3d980-94a5-11eb-9c6c-a944ef70c2a1.png)
![OcamlBefore2](https://user-images.githubusercontent.com/59281636/113478300-b315d800-94a5-11eb-8628-82ccda7abc71.png)

AFTER:
![OcamlAfter1](https://user-images.githubusercontent.com/59281636/113478315-bdd06d00-94a5-11eb-917a-c4b9d6a4450b.png)
![OCamlAfter2](https://user-images.githubusercontent.com/59281636/113478321-c45ee480-94a5-11eb-8a79-2a51781a6d80.png)

**Comment by Lovelyfin00** [comment]
Hi
I'm Loveth and an outreachy applicant. I would love to work on this

**Comment by SAY-droid427** [comment]
@patricoferris I have completed the work for this issue. But I am facing an issue. I have created a separate branch contri1 and made all the changes but when I check with git status, it shows nothing changed. Could you please help me out? This is the first time I have faced such a problem and unable to understand what to do? Should I reclone the repository and make the changes again?
![BUG1](https://user-images.githubusercontent.com/59281636/113478441-75fe1580-94a6-11eb-8f9c-9207dd68b731.png)

**Comment by SAY-droid427** [comment]
https://github.com/ocaml/ocaml.org/issues/1388#issuecomment-812859053
 Hey I have already completed the work on this issue. You won't want to copy it right? Please find some other issue

**Comment by Lovelyfin00** [comment]
Hi @SAY-droid427 
I just saw your reply now. I'll look for other issues. Thanks😊

**Comment by gurleennsidhuu** [comment]
> @patricoferris I have completed the work for this issue. But I am facing an issue. I have created a separate branch contri1 and made all the changes but when I check with git status, it shows nothing changed. Could you please help me out? This is the first time I have faced such a problem and unable to understand what to do? Should I reclone the repository and make the changes again?
> ![BUG1](https://user-images.githubusercontent.com/59281636/113478441-75fe1580-94a6-11eb-8f9c-9207dd68b731.png)

This screenshot shows that you are on branch master. Did you checkout to your new branch?

**Comment by SAY-droid427** [comment]
@gurleennsidhuu Yeah I did. I recognized the problem. I had made changes to the ocaml.org which was placed in the gitignore file. Now I have made changes to the sites/ocamlorg.css file and the changes are reflected. But I had one question. If I have to make any changes in the index.html files(not altering the contents but for changing the tags or something) which files should I modify and how?

**Comment by gurleennsidhuu** [comment]
> @gurleennsidhuu Yeah I did. I recognized the problem. I had made changes to the ocaml.org which was placed in the gitignore file. Now I have made changes to the sites/ocamlorg.css file and the changes are reflected. But I had one question. If I have to make any changes in the index.html files(not altering the contents but for changing the tags or something) which files should I modify and how?

I am not sure if this is the correct way but one of the ways can be to run `make local`. It creates html files in a new folder 'ocaml.org' and then you can open that html file in your code editor.

**Comment by gs0510** [comment]
@SAY-droid427 As mentioned on the PR here #1392, a global change is hard, do you want to pick up a specific part of the website and apply that change instead? Are you still working on this? Thanks!
